guidelines/governance/wg-governance-guidelines.md

```diff
@@ -0,0 +1,44 @@
+# Working Group Governance Guidelines
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL
+      NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and
+      "OPTIONAL" in this document are to be interpreted as described in
+      [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+## Introduction
+
+The purpose of this document is to describe working group (WG) practices and guidelines. WG members MUST adhere to these guidelines where applicable.
+
+### Relationship to SIGs
+
+All WGs MUST define their relationship to SIGs and follow the related SIGs guidelines. Deviation from related SIGs guidelines MUST be explicitly declared with stated rationale.
+
+## Leadership roles
+
+### Execution Lead
+
+The execution lead is responsible for the overall health and execution of the WG. The execution lead MUST:
+
+- Ensure that the WG goals and objectives are set
+- Set WG priorities and roadmap
+- Ensure that the WG adheres to the related SIGs guidelines
+- Ensure that the WG has sufficient resources to succeed
+- Ensure that WG progress reports are submitted on a monthly basis
+- Facilitate communication and cross-SIG collaboration if the WG is related to multiple SIGs
+- Take on technical lead role if the technical lead is not designated
+
+### Technical Lead
+
+The technical lead is responsible for the technical health and execution of the WG. The technical lead MUST:
+
+- Triage issues
+- Ensure that the WG is adhering to the related SIGs technical guidelines
+
+## Disbanding
+
+WGs MUST disband if
+- scope is completed
+- there has been no communication activity for 1 month in the defined communications channels
```

Comment:
I think it should be longer than 1 month. I would support either of the following:
- 3 months, because there is not really a bad effect of keeping the group open, other than clutter. and i think patience is appropriate here
- raise the question after 2 months (e.g. 'anyone here?')